index.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
 	globals: {
 		'__DEV__': true,
+		'process': true,
```

Comment:
I'm a bit on the fence about this change. I think we could ask developers who want to do something 'advanced' to know they're supposed to tell eslint to ignore it.  I do wonder how likely it is that developers would try to use other `process` members, though.

Reply:
The simple fact is `process.env.NODE_ENV` has always been supported (we just use `__DEV__` shortcut in our sources while babel transpiles to `process.env.NODE_ENV`. Similarly, much like `create-react-app` has, we now support `process.env.REACT_APP_*` namespace for valiable substitution. So for example, a developer could set environment variable `REACT_APP_MYAPIKEY=1234556` and in their js code write `process.env.REACT_APP_MYAPIKEY` and that variable invocation would be replaced with the value `1234556` in the the build output. A heavily useful feature as you can imagine.  For us to be expressly wanting about `process` global usage when it's an accepted feature feels antithetical to us using our own custom eslint configuration to begin with. Just my 2 cents.

As a matter of comparison, the `create-react-app` eslint config simply chooses to support most common environment globals (`browser: true, commonjs: true, es6: true, jest: true, node: true}`) and then adds eslint warnings about specific browser globals implicitly attached to the window https://github.com/facebook/create-react-app/blob/next/packages/confusing-browser-globals/index.js

Reply:
I am intrigued by this `confusing-browser-globals` package.  This seems like a useful addition to our rules.